src/deavinaSJun20/PairSumInArray.java

```diff
@@ -0,0 +1,18 @@
+package deavinaSJun20;
+
+public class PairSumInArray {
+
+	public static void main(String[] args) {
+		int[] Arr = { 2, 4, 3, 5, 7, 8, 9, -1 };
+		new PairSumInArray().pairSumInArray(Arr);
+	}
+
+	void pairSumInArray(int[] Arr) {
+		for (int outerindex = 0; outerindex < Arr.length; outerindex++) {
+			for (int innerindex = outerindex + 1; innerindex < Arr.length; innerindex++) {
+				if (Arr[outerindex] + Arr[innerindex] == 7)
```

Comment:
This is correct but to safer side always do () for mathematics operation in java, in your code line number 13

logic is correct very efficient!
Keep it up

Reply:
Sure Harsh Will keep this in mind for maths operations in future programs